test/Formik.test.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { render, fireEvent, wait } from 'react-testing-library';
 import * as Yup from 'yup';
 
-import { Formik, FormikProps, FormikConfig } from '../src';
+import { Formik, FormikProps, FormikConfig, ValueUpdate } from '../src';
```

Comment:
doesn't look like ValueUpdate is needed here

Reply:
I just pushed a commit to remove this, thanks